Tests/SDFCoreTests/MacroInitializerSorterTests.swift

```diff
@@ -1,201 +1,290 @@
 import Foundation
 import Testing
+
 @testable import SDFCore
 
 final class MacroInitializerSorterTests {
-  let sut = MacroInitializerSorter(macroName: "TestMacro")
+  struct PropertiesTests {
```

Comment:
I decided to group all the initializer properties tests and all the new assignment tests into their own structs for clarity.